routes/vault/src/components/LoanCard/LoanCard.tsx

```diff
@@ -34,6 +34,31 @@ export function LoanCard({
   const [borrowAmount, setBorrowAmount] = useState(0);
   const [repayAmount, setRepayAmount] = useState(0);
   const [withdrawCollateralAmount, setWithdrawCollateralAmount] = useState(0);
+  
+  // Hardcoded collateral step arrays
+  const borrowCollateralSteps = useMemo(() => {
+    const maxBtc = maxCollateral;
+    return [
+      { value: 0 },
+      { value: maxBtc * 0.2 },
+      { value: maxBtc * 0.4 },
+      { value: maxBtc * 0.6 },
+      { value: maxBtc * 0.8 },
+      { value: maxBtc },
+    ];
+  }, [maxCollateral]);
+  
+  const withdrawCollateralSteps = useMemo(() => {
+    const currentBtc = currentCollateralAmount;
+    return [
+      { value: 0 },
+      { value: currentBtc * 0.2 },
+      { value: currentBtc * 0.4 },
+      { value: currentBtc * 0.6 },
+      { value: currentBtc * 0.8 },
+      { value: currentBtc },
```

Comment:
We should probably take decimals into account here. Current BTC could be something like 0.001234 right? Let's make sure this renders well on those edge cases